app/renderer/util.js

```diff
@@ -283,3 +283,36 @@ export function addVendorPrefixes (caps) {
     return cap;
   });
 }
+
+
+/**
+ * check if session is run on iOS device
+ * @param  {Object}  caps  caps of session response
+ * @return {Boolean}       true if run on iOS device
+ */
+export function isIOS (caps) {
+  if (!caps) {
+    return false;
+  }
+
+  return Boolean(
+      (caps.platformName && caps.platformName.match(/iOS/i)) ||
```

Comment:
since the `platformName` cap is required for every session, we should theoretically be able to look just at that and not worry about deviceName, for both iOS and Android.

Reply:
Okay, since `platformName` cap is required, I can probably remove these util functions and move the match check out to `Session.js` since that was the only place these functions were used.

Moved here: https://github.com/appium/appium-inspector/pull/746/commits/a2618b0e892d7ea8347dcea2e3e3f904969cca42#diff-cc0a8b1e7efd7e5b9ee01a00c256283251315b2c0c18cca08911fef1c01fa0b9R530-R531